### PR TITLE
Return some impure reagents in inverse state

### LINKED ISSUE #79692: Some impure reagents cannot be cooked/do not affect the body
Round ID: [218654](https://scrubby.melonmesa.com/round/218654) (https://statbus.space/rounds/218654)

## Reproduction:
1. Add hydrogen/sodium/stable plasma to obtain a pH sufficient to reduce purity to 1-30% in future
2. Add reagents from wiki recipe
3. Move to High-performance liquid chromatography machine and see clean (not impure; not inverted) reagent
4. Drink/inject to see no impure reagent in blood and stomach and no effect

Lentslurri
image
image
image
This one "will cause the patient to move slowly" but no effect did

A similar situation with:

- Aivime
- Libitoil
- Eigenswap

### PR DESCRIPTION
**Pull request by DarkHunter7756**
## About The Pull Request
In #68385 a lot of impure reagents was converted into inverse, but not Aivime, Lentslurri, Libitoil and Eigenswap.

This PR:
1) Returns Aivime, Lentslurri, Libitoil and Eigenswap to cookable state with inverse_chem_val 
2) Makes code less outdated i think
3) Remakes Aivime (adds the ability to stack blur) 
4) Change eigenswap inverse_chem_val from 0 to 0.1 (zero purity is unreachable)
Fixes #79692
## Why It's Good For The Game
This is an effort to add unused reagents and to provide additional options for a Chemist to experiment with
## Changelog
:cl:
fix: Aivime, Lentslurri, Eigenswap and Libitoil can now be cooked again.
balance: Aivime now can stack blur effects for the soapiest game ever.
spellcheck: Lenturri description become more actual
/:cl:

### CODE REVIEW

**Comment by Higgin** [comment]
would u consider syrinifergus + oculater

**Comment by DarkHunter7756** [comment]
Other reagents will come in another pr.

**Comment by SyncIt21** [comment]
Fixes #79692. You fixed your own issue 👍

**Comment by SyncIt21** [comment]
> Fixes #79692. You fixed your own issue 👍

Make sure to add this to your PR body, you can get some GBP from this

**Review by Ghommie — CHANGES_REQUESTED** [review]

File: code/modules/reagents/chemistry/reagents/impure_reagents/impure_medicine_reagents.dm [hunk 1]
```diff
@@ -250,24 +254,20 @@ Basically, we fill the time between now and 2s from now with hands based off the
...
 	addiction_types = list(/datum/addiction/medicine = 1.5)
-	liver_damage = 0.1
-	/// blurriness at the start of taking the med
-	var/amount_of_blur_applied = 0 SECONDS
-
-/datum/reagent/impurity/aiuri/on_mob_add(mob/living/affected_mob, amount)
-	. = ..()
-	amount_of_blur_applied = creation_purity * (volume / metabolization_rate) * 2 SECONDS
-	affected_mob.adjust_eye_blur(amount_of_blur_applied)
+	var/amount_of_blur_applied = 2.5 SECONDS
```
[the remark this answers is not in the document]
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/impure_reagents/impure_medicine_reagents.dm:263]: needs to also be commented, not exactly a fan of "We use time defines but it doesnt reflect the actual time in practice".

**Comment by ZealousZeke** [comment]
Bless. 🙏

**Pull request merged**